backend/src/main/resources/liquibase/db-changelog-2.0.xml

```diff
@@ -993,4 +993,27 @@
         <tagDatabase tag="db_version_2.35"/>
     </changeSet>
 
+    <changeSet id="create-event-table" author="anandJ">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="event"/>
+            </not>
+        </preConditions>
+        <createTable tableName="event">
+            <column name="eventUUID" type="varchar(64)">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="pk_query_event_uuid"/>
+            </column>
+            <column name="eventType" type="varchar(64)"/>
+            <column name="workspaceId" type="int8"/>
+            <column name="eventMetadata" type="text"/>
```

Comment:
all other columns are in snake case. Why use camel case here?

Reply:
but you can keep camel case in java and leave snake case in liquibase, I think

Reply:
Now we have just liquibase and will create direct query so no need to add java entity anymore. SO going with snake case only. :)